spec/text-editor-spec.coffee

```diff
@@ -4018,3 +4018,37 @@ describe "TextEditor", ->
       editor.setPlaceholderText('OK')
       expect(handler).toHaveBeenCalledWith 'OK'
       expect(editor.getPlaceholderText()).toBe 'OK'
+
+  describe 'gutters', ->
+    describe 'the TextEditor constructor', ->
```

Comment:
I think we can drop this `describe` and just have a top-level `it` that says: `it "has a line-numbers gutter by default"`.